lib/hawkular/base_client.rb

```diff
@@ -175,6 +178,16 @@ def tenant_header
       headers
     end
 
+    # @!visibility private
```

Comment:
See http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md ( also for the @ in `@return` of another PR.
I can't tell if this is appropriate for this specific line. Probably not if there is not more documentation of the method.